fly/commands/builds.go

```diff
@@ -254,7 +254,7 @@ func populateTimeCells(startTime time.Time, endTime time.Time) (ui.TableCell, ui
 		durationCell.Contents = endTime.Sub(startTime).String()
 	}
 
-	if startTime == zeroTime && endTime == zeroTime {
+	if startTime == zeroTime && endTime == zeroTime || startTime == zeroTime {
```

Comment:
glad that you found this logic! but I think `(A && B) || A == A` (by set theory, or by doing the truth table), so you can just say `startTime == zeroTime` here!